Makefile

```diff
@@ -148,21 +148,21 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 	
 .PHONY: deploy-status-addon
 deploy-status-addon: kustomize ## Deploy addon to the k8s cluster specified in ~/.kube/config.
-	cd addons/status/deploy/status-sync/default && $(KUSTOMIZE) edit set image controller=${IMG}
-	$(KUSTOMIZE) build addons/status/deploy/status-sync/default | kubectl apply -f -
+	cd addons/status/statusdeploy/status-sync/default && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build addons/status/statusdeploy/status-sync/default | kubectl apply -f -
```

Comment:
No need to rename the directory `deploy`, just need to change the hub and spoke package names